CITATION.cff

```diff
@@ -0,0 +1,104 @@
+# YAML 1.2
+# More then one affiliation:
+# https://github.com/citation-file-format/citation-file-format/issues/268
+---
+title: EUCP Atlas of constrained climate projections
+keywords:
+  - climate
+  - EUCP
+repository-code: "https://github.com/eucp-project/atlas"
+license: "Apache-2.0"
+message: "If you use this software, please cite it using these metadata."
+cff-version: "1.2.0"
+authors:
+  -
+    family-names: Coppola
+    given-names: Erika
+    orcid: "https://orcid.org/0000-0001-6944-5815"
+    affiliation: The Abdus Salam International Centre for Theoretical Physics, Trieste, Italy
+  -
+    family-names: Nogherotto
+    given-names: Rita
+    orcid: "https://orcid.org/0000-0003-1718-542X"
+    affiliation: The Abdus Salam International Centre for Theoretical Physics, Trieste, Italy; Istituto Nazionale di Oceanografia e di Geofisica Sperimentale, Trieste, Italy
+  -
+    family-names: Brunner
+    given-names: Lukas
+    orcid: "https://orcid.org/0000-0001-5760-4524"
+    affiliation: ETH Zurich
+  -
+    family-names: Booth
+    given-names: Ben
+    orcid: "https://orcid.org/0000-0002-0715-2141"
+    affiliation: UK Met Office
+  -
+    family-names: Harris
+    given-names: Glen
+    orcid: "https://orcid.org/0000-0001-9259-7964"
+    affiliation: UK Met Office
+  -
+    family-names: Qasmi
+    given-names: Said
+    orcid: "https://orcid.org/0000-0002-1825-3896"
+    affiliation: Centre National de Recherches Météorologiques
+  -
+    family-names: Ballinger
+    given-names: Andrew
+    orcid: "https://orcid.org/0000-0003-3704-1976"
+    affiliation: University of Edinburgh
+  -
+    family-names: Hegerl
+    given-names: Gabriele
+    affiliation: University of Edinburgh
+    orcid: "https://orcid.org/0000-0002-4159-1295"
+  -
+    family-names: McSweeney
+    given-names: Carol
+    orcid: "https://orcid.org/0000-0001-8898-2093"
+    affiliation: UK Met Office
+  -
+    family-names: O'Reilly
+    given-names: Christopher
+    orcid: "https://orcid.org/0000-0002-8630-1650"
+    affiliation: Reading University
+  -
+    family-names: Palmer
+    given-names: Tamzin
+    orcid: "https://orcid.org/0000-0002-8889-5595"
+    affiliation: UK Met Office
+  -
+    family-names: Ribes
+    given-names: Aurélien
+    affiliation: Centre National de Recherches Météorologiques
+    orcid: 0000-0001-5102-7885
```

Comment:
```suggestion
    orcid: https://orcid.org/0000-0001-5102-7885
```

Reply:
+ add quotes to this one, like done to the others!